tests/test_code_report.py

```diff
@@ -21,7 +21,7 @@ def get_config(args: List[str]):
         from universum.configuration_support import Variations
 
         configs = Variations([dict(name="Run static pylint", code_report=True,
-            command=['python3.7', '-m', 'universum.analyzers.pylint'{''.join(args)}])])
+            command=['python{utils.PYTHON_VERSION}', '-m', 'universum.analyzers.pylint'{''.join(args)}])])
```

Comment:
I would prefer to use consistent approach:
1. always import python version variable into you module and refer to it without specifying utils module 
2. use the shorter name
3. include "python" into the variable